### PR TITLE
Add org.kde.kdeconnect

### PR DESCRIPTION
**Pull request by MrMarukesu**
maybe @grulja can be a maintainer too?

### CODE REVIEW

File: org.kde.KDEConnect.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		"/lib/pkgconfig",
+		"/lib/*.la",
+		"/lib/cmake",
+		"/share/doc",
+		"/share/man"
+    ],
+    "modules": [
+        {
+            "name": "libfakekey",
+            "buildsystem": "simple",
```
Comment (TingPing) [inline org.kde.KDEConnect.json]: Just use the default build system which is autotools?

File: org.kde.KDEConnect.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                {
+                    "type": "archive",
+                    "url": "http://downloads.yoctoproject.org/releases/matchbox/libfakekey/0.1/libfakekey-0.1.tar.bz2",
+                    "sha256": "c41042a70524895b0edf8077ae3ae1206f1a313bea5811dcc23033ea16a00194"
+                }
+            ]
+        },
+        {
+            "name": "qca",
+            "config-opts": ["-DENABLE_TESTING=OFF", "-DCMAKE_INSTALL_LIBDIR=lib"],
```
Comment (TingPing) [inline org.kde.KDEConnect.json]: Don't need to set libdir.

File: org.kde.KDEConnect.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                    "type": "git",
+                    "url": "git://anongit.kde.org/qca.git",
+                    "branch": "v2.1.3",
+                    "commit": "8c08b64c8e6094fc8cd2a240dd6f8ee58cdb724e"
+                }
+            ]
+        },
+        {
+            "name": "kdeconnect",
+            "buildsystem": "cmake-ninja",
```
Comment (TingPing) [inline org.kde.KDEConnect.json]: Set `CMAKE_BUILD_TYPE=Release`

**Comment by grulja** [comment]
I can of course co-maintain it, but maybe you want to ask @aleixpol first, he is also upstream developer of kdeconnect.

**Comment by nedrichards** [comment]
bot, build org.kde.kdeconnect

File: icons.sh [hunk 4]
```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+install -dm755 /app/share/icons/hicolor/32x32/status
+install -dm755 /app/share/icons/hicolor/scalable/status
+
+# Indicator 32px #
+
+cp /app/share/icons/hicolor/32x32/status/laptopconnected.png /app/share/icons/hicolor/32x32/status/org.kde.kdeconnect.laptopconnected.png
```
Comment (aleixpol) [inline icons.sh]: Can you elaborate on why this is needed?

**Comment by nedrichards** [comment]
the json file needs to have exactly the same name as the app-id for flathub to build it. so  `org.kde.KDEConnect.json` needs to be `org.kde.kdeconnect.json`

File: icons.sh (hunk 4 above)
Comment (MrMarukesu) [reply icons.sh]: the icons don't load here if it is not exported, and flatpak only exports icons with the app-id

**Comment by nedrichards** [comment]
bot, build org.kde.kdeconnect

File: org.kde.kdeconnect.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+        "/include",
+		"/lib/pkgconfig",
+		"/lib/*.la",
+		"/lib/cmake",
+		"/share/doc",
+		"/share/man"
+    ],
+    "modules": [
+        {
+            "name": "libfakekey",
```
Comment (nedrichards) [inline org.kde.kdeconnect.json]: this needs to be reconfigured to work on aarch64, similar to https://github.com/flathub/fr.free.Homebank/blob/91c81b69cfe98bfea5593ba6475c3b69b7308d04/fr.free.Homebank.json#L58

**Comment by nedrichards** [comment]
bot, build org.kde.kdeconnect

**Comment by barthalion** [comment]
I'm closing it due to inactivity – but please re-open it or resubmit as new pull request if you would like to see it merged in the future.

**Pull request closed without merging**